nipype/interfaces/afni/base.py

```diff
@@ -162,7 +162,7 @@ def __init__(self, **inputs):
 
         # Update num threads estimate from OMP_NUM_THREADS env var
         # Default to 1 if not set
-        os.environ['OMP_NUM_THREADS'] = str(self.num_threads)
+        self.inputs.environ['OMP_NUM_THREADS'] = str(self.num_threads)
```

Comment:
what if this changes between initialization and run. this should happen in the `_run_interface` function.